iOS_SDK/OneSignalSDK/Source/OSStateSynchronizer.m

```diff
@@ -282,16 +268,18 @@ - (void)sendOnFocusTime:(NSNumber*)totalTimeActive
                  params:(OSFocusCallParams *)params
             withSuccess:(OSMultipleSuccessBlock)successBlock
               onFailure:(OSMultipleFailureBlock)failureBlock {
-    let pushStateSyncronizer = [self getPushStateSynchronizer];
-    let emailStateSyncronizer = [self getEmailStateSynchronizer];
-    
+    let stateSyncronizer = [self getStateSynchronizers];
+    // Begin constructing the request for the external id update
```

Comment:
This comment isn't correct, this is a focus update. Recommend removing instead of updating comment as the method name already describes the request about to be made.